calico-cloud/users/create-custom-role-for-entra-id-group.mdx

```diff
@@ -0,0 +1,55 @@
+---
+description: Create custom roles for Entra ID groups.
```

Comment:
My thinking here is to write a doc that represents what a user is trying to accomplish. Which is, I think, making a role that is backed with their Entra ID group. The title you suggest seems to be an element of that larger user story.

It would be good for us to talk through, and perhaps revisit, the UX copy as it stands. "IdP group" is missing the fundamental idea of what it is: a custom role that you assign to IDP groups. It's a subset of "custom role" rather than a different thing. 

I do specify "security groups" later, but it might be worth stating explicitly that we don't support Microsoft 365 groups.